<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EzCourse</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_login">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>登录</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>241</width>
       <height>55</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_input">
      <item row="0" column="0">
       <widget class="QLabel" name="label_usrid">
        <property name="text">
         <string>学号(ID)</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_usrid</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_usrpw">
        <property name="text">
         <string>密码</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_usrpw</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_usrid">
        <property name="text">
         <string>mgr</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_usrpw">
        <property name="text">
         <string>123</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_access">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>211</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>身份</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_access">
       <item>
        <widget class="QRadioButton" name="radioButton_stu">
         <property name="text">
          <string>学生</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_mgr">
         <property name="text">
          <string>管理员</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_login">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>201812030318 程序设计课程设计</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_usrid</tabstop>
  <tabstop>lineEdit_usrpw</tabstop>
  <tabstop>radioButton_stu</tabstop>
  <tabstop>radioButton_mgr</tabstop>
  <tabstop>pushButton_login</tabstop>
 </tabstops>
 <resources>
  <include location="EzCourse.qrc"/>
 </resources>
 <connections/>
</ui>
